python/ray/dashboard/utils.py

```diff
@@ -358,6 +358,33 @@ def address_tuple(address):
     return ip, int(port)
 
 
+def node_stats_to_dict(
+    message: "ray.core.generated.node_manager_pb2.GetNodeStatsReply",
```

Comment:
why's it a string and not imported?

Reply:
I don't want to import the Python module just for type hints. Would you prefer using `from typing import TYPE_CHECKING` instead?

Reply:
update to use `TYPE_CHECKING` instead